package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "puppeteer": "^9.0.0"
```

Comment:
We should include `puppeteer` in the list of dev dependencies, then we can `npm install` it. Then we know we'll always be working with the same version of Puppeteer.